Zend/header

```diff
@@ -1,17 +0,0 @@
-/*
-   +----------------------------------------------------------------------+
-   | Zend Engine                                                          |
-   +----------------------------------------------------------------------+
-   | Copyright (c) Zend Technologies Ltd. (http://www.zend.com)           |
```

Comment:
These two files are different, they use a different license and different copyright.

Reply:
They are, yes. But isn't the Zend license header not used anymore? 

See this:
grep -Eri 'Copyright \(c\) The PHP Group' Zend

There are already recent files that are using PHP headers in the Zend directory. According to this, theoretically adding a new c file in the Zend directory should have the PHP group headers most likely according to this pattern as is happening.

Reply:
You're right. I know some of those files have been moved from PHP to Zend and use the other license header because of that. But some files like zend_weakrefs.c/h are new.

Let's double check with @zsuraski if it's okay to use the PHP license header for all new files, including inside `Zend/`.

Reply:
So, mainly here I would like to only more or less clear what to follow in the new c and h files... In the TSRM directory the pattern is more clear - PHP group headers for everything "new"... something like that.